### PR TITLE
Libaacs cleanup

### PR DESCRIPTION
**Pull request by mrdominuzq**
compiles fine under subdir and remove  "unrecognized options: --disable-examples, --disable-debug"

also i dont think the patch is needed.

### CODE REVIEW

File: packages/multimedia/libaacs/package.mk [hunk 1]
```diff
@@ -30,22 +30,14 @@ PKG_SHORTDESC="libaacs: a research project to implement the Advanced Access Cont
 PKG_LONGDESC="libaacs is a research project to implement the Advanced Access Content System specification. This research project provides, through an open-source library, a way to understand how the AACS works. This research project is mainly developed by an international team of developers from Doom9."
 
 PKG_IS_ADDON="no"
-PKG_AUTORECONF="yes"
+PKG_AUTORECONF="no"
```
Comment (stefansaraev) [inline packages/multimedia/libaacs/package.mk]: humh. why. we always do autoreconf when possible.

**Comment by mrdominuzq** [comment]
isn't only needed if configure.ac has been modified ?

**Comment by stefansaraev** [comment]
generaly yes. but in the past, some sources come with fucked up generated configure. 

ping @sraue

**Comment by sraue** [comment]
we also replace config.guess and config.sub generally which needs a autoconfig... not sure if its still needed this way, maybe for some very old packages or if new architectures are added...

whats the problem with enabled autoreconf for this package?

**Comment by mrdominuzq** [comment]
i haven't tried with autoreconf enabled, because when configure.ac wasn't modified without the patch, i thought autoreconf wasn't needed :)

**Comment by sraue** [comment]
@mrdominuzq can you try and enable if it works? if not, paste the error here

**Comment by mrdominuzq** [comment]
Yes :)

**Comment by mrdominuzq** [comment]
https://www.diffchecker.com/vlviw1am

output with/without autoreconf

**Comment by sraue** [comment]
looks good and dont breaks, so enable autoconf again and update this PR, then we will commit

**Comment by mrdominuzq** [comment]
Okay :) Will do it sunday or monday. But just to nr clear, We Walt to enable autoreconf on target packages ?

**Comment by sraue** [comment]
yes we do this, except there is no autoconf support (cmake, no configure....), or its broken (very rarly) for all packages, also host packages

**Comment by mrdominuzq** [comment]
will keep that in mind :) thanks

**Comment by mrdominuzq** [comment]
squashed :)

**Pull request merged**